src/Datagrid/Pager.php

```diff
@@ -87,11 +87,27 @@ public function computeNbResult()
         return $this->computeResultsCount();
     }
 
-    public function getResults()
+    public function getCurrentPageResults(): iterable
     {
         return $this->getQuery()->execute();
     }
 
+    /**
+     * NEXT_MAJOR: remove this method.
+     *
+     * @deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x.
+     */
+    public function getResults()
+    {
+        @trigger_error(sprintf(
+            'The "%s()" method is deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will'
```

Comment:
I don't know which of both variants we mostly use. This suggestion is made only to be consistent with the other deprecation messages in this PR.
```suggestion
            'Method "%s()" is deprecated since sonata-project/doctrine-mongodb-admin-bundle 3.x and will'
```